Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -61,7 +61,7 @@ public SiteExtensionInfo(UIPackageMetadata data)
         public SiteExtensionInfo(UISearchMetadata data)
```

Comment:
actually can remove it now after update search to use SearchLatestPackageResource.